docker/Dockerfile

```
@@ -139,11 +139,21 @@ RUN apt-get update && \
 ARG CHECKPOINT_SYNC=true
 ARG NETWORK=Mainnet
 
-ARG ZEBRA_CONF_PATH=/etc/zebra/zebrad.toml
+# Use a configurable dir and file for the zebrad configuration file
+ARG ZEBRA_CONF_PATH=/etc/zebra
```

We could change `PATH` to `DIR`
```suggestion
ARG ZEBRA_CONF_DIR=/etc/zebra
```

We could then have:
```
ZEBRA_CONF_PATH=${ZEBRA_CONF_DIR}/${ZEBRA_CONF_FILE}
```

---

I'll create a ticket to follow-up on this with a better implementation using the entrypoint.  But I do agree.